service/r2/store/stub/store.go

```diff
@@ -21,11 +21,14 @@
 package stub
```

Comment:
This implementation appears to be largely used for testing. New functions will not be implemented in this package but instead should use the mock storage implementation.